asa_parser/tests/test_parser.py

```diff
@@ -211,10 +208,4 @@ def test_show_logging_buffered(self):
         asa = ap.AsaParser(os.path.join(self.txt_path, 'show_logging_buffered.txt'))
         result = asa.show_logging_buffered()
         self.assertIn('"text":', result)
-        self.assertIn('["Aug 16 2017 15:35:37 KP-systest-admin : %ASA-4-711004: Task ran for 114 msec, Process = Unicorn Admin Handler, PC = f34bf8f4, Call stack = "', result)
```

Comment:
It looks like you didn't add back test_tech_support_detail in your fork which should have been in the file I sent you last night.